app/src/main/res/layout-land/ongoing_topic_item.xml

```diff
@@ -57,37 +57,32 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
+          style="@style/CardTitle1"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
```

Comment:
Remove `layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)